src/GUI/UI/UIPopup.cpp

```diff
@@ -1969,7 +1969,9 @@ void Inventory_ItemPopupAndAlchemy() {  // needs cleaning
             }
 
             item->uExpireTime = GameTime(pParty->GetPlayingTime() + GameTime::FromSeconds(v31));
-            item->uAttributes = alchemy_skill_level | 0x18;
+
+            // TODO(captainurist): what are we even doing here with this cast?
```

Comment:
This must be an error from refactoring where alchemy_skill_level was reused.
Im guessing it should be something like `item->uAttributes =  item->uAttributes | ITEM_TEMP_BONUS | ITEM_AURA_EFFECT_RED;`